packages/teams-js/src/public/video.ts

```diff
@@ -93,32 +140,37 @@ export namespace video {
     if (!isSupported()) {
       throw errorNotSupportedOnPlatform;
     }
-
-    registerHandler('video.newVideoFrame', (videoFrame: VideoFrame) => {
-      if (videoFrame !== undefined) {
-        frameCallback(videoFrame, notifyVideoFrameProcessed, notifyError);
-      }
-    });
+    registerHandler(
+      'video.newVideoFrame',
+      (videoFrame: VideoFrame) => {
+        if (videoFrame !== undefined) {
+          frameCallback(videoFrame, notifyVideoFrameProcessed, notifyError);
+        }
+      },
+      false,
+    );
     sendMessageToParent('video.registerForVideoFrame', [config]);
   }
 
   /**
-   * video extension should call this to notify host client that the current selected effect parameter changed.
+   * Video extension should call this to notify host client that the current selected effect changed.
    * If it's pre-meeting, host client will call videoEffectCallback immediately then use the videoEffect.
-   * If it's the in-meeting scenario, we will call videoEffectCallback when apply button clicked.
+   * If it's the in-meeting scenario, host client will call videoEffectCallback when apply button clicked.
    *
-   * @param effectChangeType - the effect change type.
+   * @param effectChangeType - The effect change type.
    * @param effectId - Newly selected effect id.
+   * @param effectParam Parameter for the selected effect.
```

Comment:
Why is an additional param besides id needed? What would this string be used for?

Reply:
It's used for avatar background